app/controllers/external_users/claims_controller.rb

```diff
@@ -17,8 +17,8 @@ class ResourceClassNotDefined < StandardError; end
 
   before_action :set_and_authorize_claim, only: %i[show edit update unarchive clone_rejected destroy summary
                                                    confirmation show_message_controls messages disc_evidence]
-  before_action :redirect_unless_editable, only: %i[edit update]
   before_action :set_form_step, only: %i[edit]
+  before_action :redirect_unless_editable, only: %i[edit update]
```

Comment:
The order of the before actions was changed as `redirect_unless_editable` now makes use of the _form_step_, so needs to have it defined before hand.